ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -291,7 +291,13 @@ objects:
 
         PXE_INITRD_URL="$(jq -r '.baseURI + .images["initramfs"].path' /var/lib/openshift-install/rhcos.json)"
         PXE_KERNEL_URL="$(jq -r '.baseURI + .images["kernel"].path' /var/lib/openshift-install/rhcos.json)"
-        PXE_OS_IMAGE_URL="$(jq -r '.baseURI + .images["metal-bios"].path' /var/lib/openshift-install/rhcos.json)"
+        if [[ $INSTALL_INITIAL_RELEASE =~ release-4.[1,2] ]]; then
```

Comment:
could be use whichever one is not empty ?? because it think the switch on version is not very stable.

Reply:
oh, that's a much better approach, thanks!

Reply:
simplified things